src/service/auth/service.ts

```diff
@@ -0,0 +1,15 @@
+import { LogInType,  RegisterAccountType } from '@/types/auth';
+import http from '../http.service'
+
+const UserService = {
+  Login: async (data:LogInType) => {
+    const response = await http.POST('auth/login', data);
+    return response;
+  },
+  Register: async (data: RegisterAccountType) => {
+    const response = await http.POST('auth/register', data);
+    return response;
+  },
+}
```

Comment:
Let's extract the login and register functions to outside this UserService object which is not needed.

We need to export only the functions directly.